badge.svg

```diff
@@ -1,121 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Won't removing this file break all of the `https://cdn.rawgit.com/continuationlabs/belly-button/master/badge.svg` links in all of the repos?

Reply:
Yes. That's the point if we're doing this, lets do it.

Reply:
I think it would be better to at least copy view-source:https://img.shields.io/badge/eslint-bellybutton-4B32C3.svg here until the links can be fixed. I know I'm not going to have the time to go through all of the repos and fix the links today.

Reply:
Oh that's a good idea, I'll do that.